esmvaltool/cmorizers/data/formatters/datasets/esacci_watervapour.py

```diff
@@ -17,19 +17,22 @@
 Modification history
```

Comment:
This script is currently not working because of the call to the cmorization function.

Reply:
Should be fixed now. I do not have acces to the dta to test, though